drivers/sensor/lis2dw12/lis2dw12.h

```diff
@@ -82,17 +82,13 @@ union axis3bit16_t {
  * struct lis2dw12_device_config - lis2dw12 hw configuration
  * @bus_name: Pointer to bus master identifier.
  * @pm: Power mode (lis2dh_powermode).
- * @int_gpio_port: Pointer to GPIO PORT identifier.
- * @int_gpio_pin: GPIO pin number connecter to sensor int pin.
  * @int_pin: Sensor int pin (int1/int2).
  */
 struct lis2dw12_device_config {
 	const char *bus_name;
 	lis2dw12_mode_t pm;
 #ifdef CONFIG_LIS2DW12_TRIGGER
-	const char *int_gpio_port;
-	uint8_t int_gpio_pin;
-	uint8_t int_gpio_flags;
+	const struct gpio_dt_spec gpio_int;
```

Comment:
Since this is not a pointer to a gpio_dt_spec, shouldn't this be `struct gpio_dt_spec gpio_int;`, i.e. no `const`?

The constness of the field is already guaranteed by the constness of the containing config struct, no?

Reply:
Yes, sure